### PR TITLE
Feature/show top profiles

### PR DESCRIPTION
**Pull request by Bluesmile82**
This PR adds the backend fetch for the top profiles to show them in the profiles root page

To try it:

1. Generate a user in the localhost admin
2. Go to: http://localhost:8081/content/admin/login
3. Add some top profiles in Contexts --> pick a context —> top profiles --> add top profile

image

### CODE REVIEW

**Comment by Bluesmile82** [comment]
Ready!

**Comment by Bluesmile82** [comment]
Conflicts solved

File: frontend/scripts/react-components/profile-root/profile-root.actions.js [hunk 1]
```diff
@@ -117,3 +119,30 @@ export const searchNodeWithTerm = searchTerm => dispatch => {
       });
     });
 };
+
+export const getTopProfiles = () => dispatch => {
+  const topProfilesURL = getURLFromParams(GET_TOP_PROFILES);
+  fetch(topProfilesURL)
```
Comment (sorodrigo) [inline frontend/scripts/react-components/profile-root/profile-root.actions.js]: could we use axios and maybe cancel if needed

File: frontend/scripts/react-components/profile-root/profile-root.container.js [hunk 2]
```diff
@@ -18,42 +18,32 @@ function mapStateToProps(state) {
         return ctx.id === selectedContext.id;
       })
     : null;
+  const parsedTopProfiles = state.profileRoot.topProfiles?.map(profile => {
```
Comment (sorodrigo) [inline frontend/scripts/react-components/profile-root/profile-root.container.js]: this seems like a selector to me

File: frontend/scripts/react-components/profile-root/profile-root.thunks.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { getTopProfiles } from 'react-components/profile-root/profile-root.actions';
+
+export const loadTopNodes = (dispatch, getState) => {
+  const state = getState();
+  if (state?.app?.profileRoot?.topProfiles) {
```
Comment (sorodrigo) [inline frontend/scripts/react-components/profile-root/profile-root.thunks.js]: I've stopped trying to do this, ends up in bugs more often than not. The browser already has a good enough cache

**Comment by sorodrigo** [comment]
I added the context info to the cards and made them random context-info

**Pull request merged**